_pkgdown.yml

```diff
@@ -0,0 +1,19 @@
+authors:
+  R Consortium:
+    href: "https://www.r-consortium.org/"
+    html: "<img src='https://www.r-consortium.org/wp-content/uploads/sites/13/2016/09/RConsortium_Horizontal_Pantone.png' height=24>"
+
+
+reference:
+- title: Shortcuts
```

Comment:
These are not really shortcuts, i.e. `r_versions()` does not give you the same information

Reply:
how would you call them?